src/DataMapper/LeaderMapper.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GroupLife\Core\DataMapper;
+
+use GroupLife\Core\Leader;
+
+class LeaderMapper
```

Comment:
Looks like this isn't related to the PR.

Reply:
I marked it as a draft, as some points were incomplete